movies-manager/src/containers/header/header.component.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from "react";
+import { Logo } from "../../components";
+import { Wrapper } from "../wrapper/wrapper.component";
+import { Modal } from "../modal/modal.component";
+import { FormPage } from "../form-page/form-page.component";
+import { IMovie, TNullable } from "../../types/types";
+import "./header.component.scss";
+
+export interface IHeaderProps {
+    pageName: PageName;
+    onAddBtnClick: (newMovie: IMovie) => void;
+};
+
+export enum PageName {
+    Main = 'Main',
+    AddForm = 'AddForm',
+    EditForm = 'EditForm',
+}
+
+const blockName = "header";
+
+interface IMainState {
+    isDialogOpen: boolean;
+    pageName: PageName;
+}
+
+export class Header extends Component<IHeaderProps, IMainState> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            isDialogOpen: false,
+            pageName: this.props.pageName,
+        }
+    }
+
+    showModal = () => {
+        this.setState({ isDialogOpen: true });
+	    document.body.classList.add('overflow-hidden');
```

Comment:
This is dirty. Find another way of how to prevent appearance of a scrollbar without document.body.classList.

Reply:
apply class to root element